src/sc2renderer/sc2_renderer.cc

```diff
@@ -135,8 +135,13 @@ void Matrix8BPPPlayers(const char* bytes, int w_mat, int h_mat, int off_x, int o
 void ImageRGB(const char* bytes, int width, int height, int off_x, int off_y) {
     assert(renderer_);
     assert(window_);
-
+#ifdef linux
+    SDL_Surface *surface = SDL_CreateRGBSurfaceWithFormatFrom((void *) bytes, width, height, 24, 3 * width,
```

Comment:
Style: Please, follow style used in this file. Currently pointer declaration style is different.